neo4j/_async/driver.py

```diff
@@ -347,7 +371,14 @@ async def get_server_info(self):
 
         .. versionadded:: 5.0
         """
-        async with self.session() as session:
+        if config:
+            experimental_warn(
+                "All configuration key-word arguments to "
+                "verify_connectivity() are experimental. They might be "
+                "changed or removed in any future version without prior "
+                "notice."
+            )
+        async with self.session(**config) as session:
```

Comment:
We are missing tests for checking if the experimental warning is also triggered in the `get_server_info`. The PR description can also mentioned the changes in the `get_server_info`